Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -502,6 +501,38 @@ theorem norm_le_mul_norm_add (u v : E) : ‖u‖ ≤ ‖u * v‖ + ‖v‖ :=
     ‖u‖ = ‖u * v / v‖ := by rw [mul_div_cancel_right]
     _ ≤ ‖u * v‖ + ‖v‖ := norm_div_le _ _
 
+/-- An analogue of `norm_le_mul_norm_add` for the multiplication from the left. -/
+@[to_additive]
+theorem norm_le_mul_norm_add' (u v : E) : ‖v‖ ≤ ‖u * v‖ + ‖u‖ :=
```

Comment:
Sorry to keep dragging this PR on... Can you add the `nnnorm` version of this too, to match `nnnorm_le_add_nnnorm_add`?

Reply:
The proof should be literally `norm_le_mul_norm_add' u v` I think

Reply:
no problem, thanks for comments!
done